src/apps/activities/errors.py

```diff
@@ -0,0 +1,19 @@
+import apps.shared.errors as general_errors
+
+
+class ActivitiesError(general_errors.BaseError):
+    def __init__(self, message="", *args) -> None:
+        fallback = "Activity service error"
+        super().__init__(message or fallback, *args)
+
+
+class ActivityAlreadyExist(general_errors.ValidationError):
+    def __init__(self, *args) -> None:
+        message = "Activity already exists"
+        super().__init__(message, *args)
+
+
+class ReusableItemChoiceAlreadyExist(general_errors.ValidationError):
+    def __init__(self, *args) -> None:
+        message = "Reusable item choice already exist"
+        super().__init__(message, *args)
```

Comment:
Maybe, we should provide an argument for these error classes in order to provide information about the field that has to be unique?